.github/workflows/ci.yaml

```diff
@@ -0,0 +1,25 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  keycloak-build-docker:
+    name: Build Keycloak images
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Build Keycloak.X image
+        uses: docker/build-push-action@v2
```

Comment:
Please pin to commit SHA, as versions can be modified, a review of the actions also has to be done before we add additional community maintained GH actions

Reply:
@stianst Docker images are now pointing to stable SHAs, regarding the GH Action it's now only using `checkout` at the very same version as the main repo `keycloak/keycloak`.
Do you think this is enough?

Reply:
@stianst included the specific `sha` even for the checkout action:
https://github.com/keycloak/keycloak-containers/pull/340/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR13

Reply:
Pinpoint only the GH Actions, not the Docker images (as we don't have a way to keep those updated atm)

Reply:
@stianst reverted this change as the only action used here is provided by GH now.